### ISSUE TITLE
Rarity/include message about missing attributes in output instead of throwing error

### ISSUE THREAD

**Issue by kacper-koza-arianelabs** [hashgraph/hedera-nft-utilities#26]
### Problem

The problem occurs when I use `calculateRarity` function. Currently, when one of my JSON files doesn't contain `attributes` property, the package throws an error
![image](https://github.com/hashgraph/hedera-nft-utilities/assets/129764375/c3dc8e3c-4de4-4b61-b02a-e8fdcb1b7324)

We can even regard it as a bug because the metadata set without attributes is still valid.

### Solution

Include a message about a failed attempt to calculate rarity in the output: 
```
[
  {
      totalRarity: "1.00" // metadata with attributes
  },
  {
      totalRarity: 'Cannot calculate - no attributes' // metadata without attributes
  }
]
```

or 

```
[
  {
      totalRarity: "1.00",
      validAttributes: true
  },
  {
      totalRarity: undefined
      validAttributes: false
  }
]
```

Those are only my suggestions. All reasonable solutions are fine.

**Comment by michielmulders** [comment]
After consideration, I don't think we should skip files during the rarity calculation when it's missing attributes. If this happens, something is clearly wrong with the collection and I would make the final rarity calculation invalid. From a code perspective, it's better to throw an error, which you can catch, and not calculate rarity for such a use case. 

Here's the PR adding the error: https://github.com/hashgraph/hedera-nft-utilities/pull/29

**Comment by michielmulders** [comment]
Closed by #29
